Frameworks/CoreFoundation/Locale.subproj/CFNumberFormatter.c

```diff
@@ -786,7 +786,7 @@ void CFNumberFormatterSetProperty(CFNumberFormatterRef formatter, CFStringRef ke
 #if U_ICU_VERSION_MAJOR_NUM >= 55
         __CFGenericValidateType(value, CFNumberGetTypeID());
         CFNumberGetValue((CFNumberRef)value, kCFNumberSInt32Type, &n);
-        __cficu_unum_setContext(formatter->_nf, n, &status);
+        __cficu_unum_setContext(formatter->_nf, (UDisplayContext)n, &status);
```

Comment:
let's not static cast, this is a C file and upstreamable.

---
In reply to: [71390074](https://github.com/Microsoft/WinObjC/pull/668#discussion_r71390074) [](ancestors = 71390074)